service/collector/elb.go

```diff
@@ -0,0 +1,136 @@
+package collector
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/giantswarm/aws-operator/client/aws"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	AccountLabel = "account"
+)
+
+var (
+	elbsDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "elb_instance_out_of_service_count"),
+		"Gauge about ELB instances being out of service.",
```

Comment:
Is this actually a gauge, if you just build the const metric on the fly? :philosoraptor:

Reply:
I am not sure. I was all copy/paste and others use it as gauge already.